### PR TITLE
SUR-310 - UAT Fixes

### PR DESCRIPTION
**Pull request by ravindra114**
### Checklist:

-   [ ] My code is tested
-   [ ] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [ ] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by jaieds — CHANGES_REQUESTED** [review]
Can we set the full width for preset buttons? Currently, the today button looks different from other buttons because of the width and not looking consistent.

https://d.pr/i/LoqCIK

File: src/components/input/input.jsx [hunk 1]
```diff
@@ -175,8 +175,15 @@ const InputComponent = (
 					<Upload />
 				</div>
 			);
+		} else if ( ! suffix ) {
+			return null;
 		}
```
Comment (jaieds) [inline src/components/input/input.jsx:180]:
```suggestion
		} 

		if ( ! suffix ) {
			return null;
		}
```

**Review by jaieds — CHANGES_REQUESTED** [review]
- Menu item overflowing because of min-width.

https://d.pr/i/aGqfIs

File: src/components/menu-item/menu.stories.js [hunk 2]
```diff
@@ -49,12 +49,28 @@ export default {
 			description: 'Content inside the Menu Item',
 			control: { type: 'text' },
 		},
+		showArrowOnHover: {
```
Comment (jaieds) [inline src/components/menu-item/menu.stories.js:52]: Could you write an example of the use of this prop to understand it better? Currently, it is not working in the storybook's existing example code.

Comment (ravindra114) [reply src/components/menu-item/menu.stories.js:52]: Yes it is there in storybook and readme.md

**Comment by ravindra114** [comment]
> * Menu item overflowing because of min-width.
> 
> https://d.pr/i/aGqfIs

Yes, min-width is requirement asked by Akbar, so we will need to pass a min-width for spectra story
In spectra story, we can pass our custom min-width class, by default it will be min-w-64

example:
`<Menu size="md" className="w-full p-0 gap-4 **min-w-56**">`

File: src/components/input/input.jsx (hunk 1 above)
Comment (ravindra114) [reply src/components/input/input.jsx]: done

**Review by jaieds — CHANGES_REQUESTED** [review]
Could you please add fade-in & fade-out animation to the showArrowOnHover? Currently, it's showing/hiding the icon instantly. To improve the UX we can use framer motion to add that animation. It will be 2-3 lines of code.

https://d.pr/i/nen8yJ

File: src/components/menu-item/menu.stories.js (hunk 2 above)
Comment (jaieds) [reply src/components/menu-item/menu.stories.js:52]: We could mention in the description that when the menu items section is collapsed, the icon will be hidden and will show on hover if this option is enabled.

**Comment by ravindra114** [comment]
> Could you please add fade-in & fade-out animation to the showArrowOnHover? Currently, it's showing/hiding the icon instantly. To improve the UX we can use framer motion to add that animation. It will be 2-3 lines of code.
> 
> https://d.pr/i/nen8yJ

done

**Review by jaieds — APPROVED** [review]

**Comment by jaieds** [comment]
@ravindra114, we can add more details about the `showArrowOnHover` prop. Like when it will work and how? The current description is not clear enough about its purpose.

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**